support/templates/agenda-prepublish.hbs

```diff
@@ -0,0 +1,29 @@
+<div class="agendapunten" prefix={{prefixes}}>
```

Comment:
it seems this is not working as expected, at least I get the following rendered in the frontend:
```
<div class="agendapunten" prefix="eli:" http:="" data.europa.eu="" eli="" ontology#="" prov:="" www.w3.org="" ns="" prov#="" mandaat:="" data.vlaanderen.be="" mandaat#="" besluit:="" besluit#="" ext:="" mu.semte.ch="" vocabularies="" ext="" person:="" person#="" persoon:="" persoon#="" dateplugin:="" say.data.gift="" manipulators="" insertion="" besluittype:="" https:="" id="" concept="" besluittype="" dct:="" purl.org="" dc="" terms="" mobiliteit:="" mobiliteit#="" lblodmow:="" data.lblod.info="" mobiliteit="">```

Reply:
Probably also because of the missing quotes, should be solved now but i will check